static/src/javascripts/projects/common/modules/commercial/contributions-liveblog-utilities.js

```diff
@@ -128,6 +129,21 @@ const addEpicToBlocks = (
     });
 };
 
+export const setupRemoteEpicInLiveblog = (
+    Component,
+    props,
+) => {
+    const blocks = getBlocksToInsertEpicAfter();
```

Comment:
`getBlocksToInsertEpicAfter` returns an array of blocks but this is actually a legacy feature.
I don't want to change this in this PR because it's tied to removing all the rest of the epic rendering logic, which I want to do in a follow-up PR.

Reply:
yeah makes sense 👍